packages/reference/src/resources/ExternalResourceCard/ExternalResourceCard.tsx

```diff
@@ -86,7 +86,6 @@ ExternalResourceCardDescription.displayName = 'ExternalResourceCardDescription';
 export function ExternalResourceCard({
   info,
   isClickable,
-  onEdit,
```

Comment:
should we remove it from `ExternalResourceCardProps` as we are not using it any more?